spec/models/dataset_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Dataset do
+  subject { described_class.new(uri) }
+  let(:uri) { "http://localhost:3000#dataset" }
```

Comment:
Rather than hard code this value, should we use "Setting.root_uri" (which should return what is configured for the test environment)?